sqlmesh/utils/pydantic.py

```diff
@@ -262,18 +265,83 @@ def positive_int_validator(v: t.Any) -> int:
     return v
 
 
+def _get_expressions(
+    v: t.Any,
+    values: t.Union[t.Dict, ValidationInfo],
+    coerce_func: t.Optional[t.Callable[[t.Any], t.Any]] = None,
+) -> t.List[exp.Expression]:
```

Comment:
is this newly added? why did these get added?

Reply:
Working on splitting out validators for creating Expressions and Columns

Reply:
did this need to be done in this pr? it's hard for me to re-review things with big new changes like this

Reply:
I thought this is what you wanted?